plasmapy/analysis/sig_analysis.py

```diff
@@ -0,0 +1,46 @@
+"""_summary_
+"""
+
+import numpy as np
+import scipy.io as spio
+import scipy.integrate as sp
+from scipy import signal
+from warnings import warn
+
+
+def remove_offset(data_array: np.ndarray, start_idx: int, end_idx: int) -> np.ndarray:
+    """_summary_ : Function which preforms mean subtraction
+
+    Args:     
+       data_array (array): array containing values from which to mean sub
+       start_idx (int): idx of where to start obtaining mean from
+       end_idx (int): idx of where to stop obtaining mean from
+    Returns:
+       data_array (array): array after mean_sub is preformed 
+    """
```

Comment:
We use the [`numpydoc` style](https://numpydoc.readthedocs.io/en/latest/format.html) for our documentation.  Can you please look into converting your docstrings?

You can also refer to our [documentation guide](https://docs.plasmapy.org/en/stable/contributing/doc_guide.html).